hpat/hiframes/pd_series_ext.py

```diff
@@ -974,28 +974,27 @@ def array_attribute_attachment(self, ary):
 
 
 explicit_binop_funcs = {
-    operator.add: 'add',
-    operator.sub: 'sub',
-    operator.mul: 'mul',
-    operator.truediv: 'div',
-    operator.truediv: 'truediv',
-    operator.floordiv: 'floordiv',
-    operator.mod: 'mod',
-    operator.pow: 'pow',
-    operator.lt: 'lt',
-    operator.gt: 'gt',
-    operator.le: 'le',
-    operator.ge: 'ge',
-    operator.ne: 'ne',
-    operator.eq: 'eq',
+    'add': operator.add,
```

Comment:
Do you think it will faster? Or why you changed the order?

Reply:
No, the order was changes because there were two values with the same key in original explicit_binop_funcs:
    operator.truediv: 'div',
    operator.truediv: 'truediv',

Hence iterating over (key, value) pairs didn't work and only one (truediv) Series method was generated, so Series.div didn't work at all - it failed with:
numba.errors.TypingError: Failed in hpat mode pipeline (step: nopython frontend)
Unknown attribute 'div' of type series(int64, array(int64, 1d, C), none, False)